cmd/admin-update.go

```diff
@@ -30,13 +30,20 @@ import (
 	"github.com/minio/pkg/console"
 )
 
+var adminUpdateFlags = append([]cli.Flag{
+	cli.BoolFlag{
+		Name:  "yes, y",
+		Usage: "Confirms the server update",
+	},
+}, subnetCommonFlags...)
```

Comment:
not sure why this is needed subnetCommonFlags, can you explain ?

Reply:
that's a mistake on my part, fixed